radio/src/gui/colorlcd/radio_sdmanager.cpp

```diff
@@ -347,6 +347,103 @@ void RadioSdManagerPage::build(FormWindow * window)
                     deviceFirmwareUpdate);
                 dialog->flash(getFullPath(name));
               });
+            } else if (!READ_ONLY() && !strcasecmp(ext, FRSKY_FIRMWARE_EXT)) {
+              FrSkyFirmwareInformation information;
+              if (readFrSkyFirmwareInformation(getFullPath(name), information) == nullptr) {
+  #if defined(INTERNAL_MODULE_PXX1) || defined(INTERNAL_MODULE_PXX2)
+                menu->addLine(STR_FLASH_INTERNAL_MODULE, [=]() {
```

Comment:
Seems all the flashing functions are the same, we should use a function that returns the proper function pointer with just that one param being different.